pkg/common/aws/session.go

```diff
@@ -48,6 +50,13 @@ func (CcsAwsSession *ccsAwsSession) GetAWSSessions() error {
 		CcsAwsSession.session, err = session.NewSessionWithOptions(options)
 		CcsAwsSession.iam = iam.New(CcsAwsSession.session)
 		CcsAwsSession.ec2 = ec2.New(CcsAwsSession.session)
+		if viper.GetString(config.AWSAccount) != "" {
+			CcsAwsSession.accountId = viper.GetString(config.AWSAccount)
+		} else if os.Getenv("AWS_ACCOUNT_ID") != "" {
+			CcsAwsSession.accountId = os.Getenv("AWS_ACCOUNT_ID")
+		} else {
+			CcsAwsSession.accountId = ""
+		}
```

Comment:
```suggestion
		CcsAwsSession.accountId = viper.GetString(config.AWSAccount)
```

We should be able to just have one call to viper which will get the aws account id either from custom config input or environment variable. As viper will get the value from environment variable named `AWS_ACCOUNT` [here](https://github.com/openshift/osde2e/blob/main/pkg/common/config/aws_config.go#L29).

If we also want to take into account the environment variable named `AWS_ACCOUNT_ID`, we should just update the list of environment variables viper can get the value from:

```go
viper.BindEnv(AwsAccount, "AWS_ACCOUNT", "AWS_ACCOUNT_ID")
```

If none is set, viper will have the default value be an empty string.